config/vault_namespace/config.go

```diff
@@ -0,0 +1,13 @@
+package vault_namespace
+
+import "github.com/upbound/upjet/pkg/config"
+
+// ConfigureNamespace configures the namespace resource.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("vault_namespace", func(r *config.Resource) {
+		r.Kind = "VaultNamespace"
+		r.ShortGroup = "vault"
+		r.UseAsync = true
```

Comment:
nit: The value of the configuration is `true` in upjet by default. If you do not specify explicitly here, still it should be `true`.

Reply:
Done 👍